src/include/libpmemobj/action_base.h

```diff
@@ -72,9 +72,12 @@ struct pobj_action {
 };
 
 #define POBJ_MAX_ACTIONS 60
+#define POBJ_ACTION_XRESERVE_VALID_FLAGS POBJ_XALLOC_CLASS_MASK
```

Comment:
Please add support for POBJ_XALLOC_ZERO.

Reply:
It doesn't make sense, it's not that this is intended to do. If you want to zero it, just memset it after reserving.

Or saying differently, it would needlessly complicate the code.

Reply:
The same can be said about zalloc/xalloc, but we still support it.

Reply:
xalloc/zalloc already have the code to support that and it's just a variant. In this case, I would have to create a special case just to memset it.

Reply:
I thought about it and it's not that much of a cost and I can reuse some of the xalloc code.